dockstore-webservice/src/test/resources/gatk-sv-clinical/README.md

```diff
@@ -0,0 +1,36 @@
+# gatk-sv-clinical  
```

Comment:
Should document which repo this is from and what commit/tag. 
We may want to update this in the future and it won't be obvious due to size what this came from

Reply:
Fixed in upcoming push